src/components/Signup.js

```diff
@@ -161,9 +168,16 @@ const Signup = () => {
 							Sign Up
 						</Button>
 						<Grid container justify="flex-end">
+							<Grid item xs>
+								<div href="#" variant="body2">
+									<input type="checkbox" checked={check} onChange={AdminCheck}/>
+										Login As Admin
+								</div>
+							</Grid>
```

Comment:
Move the checkbox before the Sign Up button
Edit label to say Signup as Admin instead of Login as Admin